isos_post.py

```diff
@@ -9,10 +9,14 @@
 
 def get_job_name(host, job_id):
     cmd = '/usr/bin/openqa-client --json-output --host {} jobs/{}'.format(host, job_id)
-    print('getting job name : {} \n'.format(cmd))
+    print('getting job name : {}'.format(cmd))
     o_json = json.loads(subprocess.check_output(cmd, shell=True))
     return o_json['job']['name']
 
+def set_job_priority(host, job_id, priority):
+    cmd = 'openqa-client --host {} jobs/{} put --json-data \'{{"priority": {} }}\''.format(host, job_id, priority)
+    print('setting job priority : {}'.format(cmd))
+    subprocess.check_output(cmd, shell=True);
```

Comment:
because we want to set priorities for several jobs and probably don't want to interrupt in case one job will produce some error , we might to think about wrap this section in some try/catch block so we would just error and not fully stop in case something will pop up here

Reply:
ok